{"name": "整体增删改查用户", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: '../api/wework.yaml'", "trace": "self = <WechatAPI.testcase.test_work.TestWework object at 0x000001F1157BD6A0>\nuserid = 'userid6', name = '名字6', mobile = '13800000006'\n\n    @allure.story(\"通讯录管理\")\n    @allure.title(\"整体增删改查用户\")\n    @pytest.mark.parametrize(\"userid,name,mobile\",test_data())\n    def test_wechat(self,userid,name,mobile):\n    \n        # userid=\"zhangsan90\"\n        # name=\"张三思0\"\n        # mobile = \"13290900011\"\n        try:\n>           assert \"created\"==Wework().test_creat_member(userid,name,mobile)[\"errmsg\"]\n\nWechatAPI\\testcase\\test_work.py:43: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <WechatAPI.api.we_work.Wework object at 0x000001F1157E80F0>\n\n    def __init__(self):\n        self.token = Util().token() #不要忘了Util的()\n        self.params[\"token\"]=self.token #要记得替换token的变量\n    \n>       with open(\"../api/wework.yaml\",encoding=\"utf-8\")as f:\nE       FileNotFoundError: [Errno 2] No such file or directory: '../api/wework.yaml'\n\nWechatAPI\\api\\we_work.py:27: FileNotFoundError"}, "parameters": [{"name": "userid", "value": "'userid6'"}, {"name": "name", "value": "'名字6'"}, {"name": "mobile", "value": "'13800000006'"}], "start": 1647486672259, "stop": 1647486672597, "uuid": "91190a83-daf8-436f-a9df-a68716c92a8b", "historyId": "881345649f21f74ca32a0e470e7792cd", "testCaseId": "6fbae5519000b9212e1162caf7211d2e", "fullName": "WechatAPI.testcase.test_work.TestWework#test_wechat", "labels": [{"name": "story", "value": "通讯录管理"}, {"name": "parentSuite", "value": "WechatAPI.testcase"}, {"name": "suite", "value": "test_work"}, {"name": "subSuite", "value": "TestWework"}, {"name": "host", "value": "ZXJ"}, {"name": "thread", "value": "58484-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "WechatAPI.testcase.test_work"}]}